src/databricks/labs/ucx/runtime.py

```diff
@@ -67,6 +68,58 @@ def inventorize_mounts(cfg: MigrationConfig):
     mounts.inventorize_mounts()
 
 
+@task("assessment", depends_on=[inventorize_mounts, crawl_tables])
+def inventorize_external_locations(cfg: MigrationConfig):
+    """In this part of the assessment, we're going to look up the location of all the tables.
+    Locations that use
+    Mount Points will be looked up. We will then find out the External Locations required for a proper migration and
+    save these to the `$inventory.external_locations` Table.
+    The logic this assessment follow is extract all the locations for tables that are not using DBFS
+    (we capture the locations for tables using mount points).
+    We then scan all the locations and find the common folders that accommodate them.
+    These external_locations will be created in a later stage before the table can be migrated.
+    """
+    ws = WorkspaceClient(config=cfg.to_databricks_config())
+    assess = AssessmentToolkit(ws, cfg.inventory_database, RuntimeBackend())
+    assess.generate_ext_loc_list()
+
+
+@task("assessment", depends_on=[setup_schema])
+def inventorize_jobs(cfg: MigrationConfig):
+    """This part scan through all the jobs and locate ones that are not compatible with UC.
+    It looks for:<br>
+    <ol>
+    <li>Clusters with DBR version earlier than 11.3<br>
```

Comment:
Looks like an invalid html to me. Perhaps you wanted </li>. Though, do you **really want** your tasks to appear differently than others in the readme?

Reply:
I want all of them to look nice.

Reply:
This is not a valid HTML. Either make a proper closing tag for LI or make it all look consistent with the rest of the documentation strings. 

On the other hand, Markdown has it's own way of showing lists. We want a consistent user-facing documentation and this startles the eye. If you want to improve the readme notebook display style - make a separate PR.